tests/jballerina-integration-test/src/test/resources/http/src/httpservices/11_http_redirects.bal

```diff
@@ -252,6 +253,78 @@ service testRedirect on serviceEndpoint3 {
             io:println("Connector error!");
         }
     }
+
+    @http:ResourceConfig {
+        methods: ["GET"],
+        path: "/doPost"
+    }
+    resource function PostClearText(http:Caller caller, http:Request request) {
+        http:Client endPoint4 = new("http://localhost:9103", endPoint4Config );
+        http:Request req = new;
+        req.setHeader("proxy-authorization", "Basic YWxhZGRpbjpvcGVuc2VzYW1l");
+        req.setTextPayload("Payload redirected");
+        var response = endPoint4->post("/redirect1/handlePost", req);
+        if (response is http:Response) {
+            var value = response.getTextPayload();
+            if (value is string) {
+                value = value + ":" + response.resolvedRequestedURI;
+                checkpanic caller->respond(<@untainted> value);
+            } else {
+                io:println("Payload error!");
+            }
+        } else {
+            io:println("Connector error!");
+        }
+    }
+
+    @http:ResourceConfig {
+        methods: ["GET"],
+        path: "/doSecurePut"
+    }
+    resource function testSecurePut(http:Caller caller, http:Request request) {
+        http:Request req = new;
+        req.setHeader("proxy-authorization", "Basic YWxhZGRpbjpvcGVuc2VzYW1l");
```

Comment:
```suggestion
        req.setHeader("Proxy-Authorization", "Basic YWxhZGRpbjpvcGVuc2VzYW1l");
```

Reply:
Keeping 1 in lower case to get it tested :)